djangosaml2/middleware.py

```diff
@@ -67,8 +67,8 @@ def process_response(self, request, response):
                         max_age=max_age,
                         expires=expires, domain=settings.SESSION_COOKIE_DOMAIN,
                         path=settings.SESSION_COOKIE_PATH,
-                        secure=settings.SESSION_COOKIE_SECURE or None,
-                        httponly=settings.SESSION_COOKIE_HTTPONLY or None,
-                        samesite=None
+                        secure=settings.SESSION_COOKIE_SECURE,
+                        httponly=settings.SESSION_COOKIE_HTTPONLY,
+                        samesite=settings.SESSION_COOKIE_SAMESITE
```

Comment:
SameSite behaviour should be covered by unit tests, using an SP and an idp with different doman name and a pure web client, like python requests